dashboard/app/controllers/scripts_controller.rb

```diff
@@ -45,6 +45,16 @@ def show
     @section = current_user&.sections&.find_by(id: params[:section_id])&.summarize
     sections = current_user.try {|u| u.sections.where(hidden: false).select(:id, :name, :script_id, :course_id)}
     @sections_with_assigned_info = sections&.map {|section| section.attributes.merge!({"isAssigned" => section[:script_id] == @script.id})}
+
+    if current_user.levelbuilder?
+      @script.stages.each do |stage|
```

Comment:
What's the intended behavior here if more than one stage has a `visible_after`?

Reply:
@davidsbailey what was your thoughts here?

Reply:
It now concats multiple warnings together